kartograf/rpki/fetch.py

```diff
@@ -1,6 +1,7 @@
 import subprocess
 import sys
 
+from concurrent.futures import ThreadPoolExecutor, as_completed
```

Comment:
Removing in one commit and then re-adding in the next is a bit annoying to review. You could squash both commits and that would be a bit easier.